src/Sparrow/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -43,6 +43,25 @@ public static void WriteArray(this AbstractBlittableJsonTextWriter writer, strin
             writer.WriteEndArray();
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void WriteArray(this AbstractBlittableJsonTextWriter writer, JsonOperationContext context, string name, IEnumerable<DynamicJsonValue> items)
```

Comment:
don't we have something like this already?

Reply:
https://github.com/ravendb/ravendb/pull/19653#discussion_r1871317012

Reply:
Removed and used existing overload